easybuild/easyblocks/generic/pythonpackage.py

```diff
@@ -687,16 +687,18 @@ def sanity_check_step(self, *args, **kwargs):
             }
 
         # make sure 'exts_filter' is defined, which is used for sanity check
-        if self.multi_python:
-            # when installing for multiple Python versions, we must use 'python', not a full-path 'python' command!
```

Comment:
please retain this comment below, so we're not left wondering why replacing `'python'` with the full path is not done when using multi-Python